legend-engine-xt-relationalStore-executionPlan/pom.xml

```diff
@@ -99,6 +99,10 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-xt-relationalStore-executionPlan-connection-authentication</artifactId>
         </dependency>
+        <dependency>
```

Comment:
should the scope of this dep be runtime?

Reply:
In fact, no dependency is needed, cause binding happens in server. Removed.